psycopg_c/psycopg_c/types/uuid.pyx

```diff
@@ -0,0 +1,138 @@
+cimport cython
+
+from types import ModuleType
+from cpython.bytes cimport PyBytes_AsString
+from cpython.long cimport PyLong_FromUnsignedLongLong
+
+cdef extern from "Python.h":
+    # PyUnicode_AsUTF8 was added to cpython.unicode in 3.1.x but we still
+    # support 3.0.x
+    const char *PyUnicode_AsUTF8(object unicode) except NULL
+
+
+uuid: ModuleType | None = None
```

Comment:
I might have to rethink this. From previous experiences with psycopg2, storing Python objects in static C variables is dangerous for environments using multiple interpreters (such as Apache mod_python).

Maybe it would be better to drop this module var and just have `import uuid` in the `cinit`s. Importing an already imported module is a fast operation anyway.